src/abstractdataframe/abstractdataframe.jl

```diff
@@ -374,28 +374,35 @@ end
 Summarize the columns of an AbstractDataFrame
 
 ```julia
-describe(df::AbstractDataFrame)
-describe(io, df::AbstractDataFrame)
+describe(df::AbstractDataFrame; colstats = [:mean, :min, :median, :max, :Nmissing, :datatype])
+describe(io, df::AbstractDataFrame; colstats = [:mean, :min, :median, :max, :Nmissing, :datatype])
```

Comment:
I'd use `stats`, `:nmissing` and `:eltype`.

Reply:
Fixed all this, and changed `Nunique` to `nunique` as well.